test/cri-containerd/policy_test.go

```diff
@@ -1195,3 +1200,62 @@ func Test_ExecInContainer_WithPolicy(t *testing.T) {
 		})
 	}
 }
+
+func Test_ExecInUVM_WithPolicy(t *testing.T) {
+	requireFeatures(t, featureLCOWIntegrity)
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	for _, tc := range []struct {
+		execInUVMConfig  securitypolicy.ExternalProcessConfig
+		execInUVMRequest []string
+		shouldFail       bool
+	}{
+		{
+			execInUVMConfig: securitypolicy.ExternalProcessConfig{
+				Command:          []string{"ls"},
+				WorkingDir:       "/",
+				AllowStdioAccess: true,
+			},
+			execInUVMRequest: []string{"ls"},
+			shouldFail:       false,
+		},
+		{
+			execInUVMConfig: securitypolicy.ExternalProcessConfig{
+				Command:          []string{"ls"},
+				WorkingDir:       "/",
+				AllowStdioAccess: true,
+			},
+			execInUVMRequest: []string{"ls", "-l"},
+			shouldFail:       true,
+		},
+	} {
+		t.Run(fmt.Sprintf("ExecInUVM_ShouldFail_%t", tc.shouldFail), func(t *testing.T) {
```

Comment:
Ttest name stutters: `Test_ExecInUVM_WithPolicy/ExecInUVM_ShouldFail_false` (or `_true`).
Can probably shorten subtest name to `fmt.Sprintf("ShouldFail_%t", tc.shouldFail)`